WooCommerce/Classes/Analytics/WooAnalyticsEvent.swift

```diff
@@ -1614,13 +1614,20 @@ extension WooAnalyticsEvent {
         }
 
         enum Name: String {
-            case todayStats = "today-stats"
+            case todayStats = "stats-today"
```

Comment:
When we add time interval configuration (week/month/year) later, we can update it to `stats-\(timeIntervalName)`

Reply:
Another important configuration ideas to consider adding (and logging in analytics) in the future:
- Store (so user add same widget for different stores)
- Metric (select number of orders or visitors instead of total revenue)

It could look like:
```
stats-visitors-week-accessoryCircular
```

Reply:
I'm not sure if I follow the idea here, what would be the problem to have future widgets as `weekly-stats` and `year-stats`?

Also keep in mind that if we change this identifier, we should also change it on Android, and keep the old name into account for the first widget analytics report.

Reply:
> what would be the problem to have future widgets as weekly-stats and year-stats

That would be the same widget id, just with some additional configuration attached. But sure, we can log anything, including "weekly-stats and year-stats".

I haven't thought about same naming on Android side. Then probably we should revert 8c3050a?
What about familyName? `today-stats` became `today-stats-systemMedium`. We can make exception to it and log without a suffix to keep alignment with Android.

Reply:
Revert pushed.